dev.pulsar-edit.Pulsar.json

```diff
@@ -0,0 +1,88 @@
+{
+    "app-id": "dev.pulsar-edit.Pulsar",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "22.08",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "pulsar",
+    "rename-desktop-file": "pulsar.desktop",
+    "rename-icon": "pulsar",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--socket=ssh-auth",
+        "--share=network",
+        "--device=all",
+        "--filesystem=host",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.secrets",
+        "--talk-name=org.gtk.vfs.*",
+        "--env=ELECTRON_TRASH=gio",
+        "--env=TMPDIR=/var/tmp",
+        "--filesystem=xdg-data/Trash",
+        "--filesystem=xdg-config/kdeglobals:ro",
```

Comment:
why do you need these filesystem permissions?

Reply:
Well I thought that Pulsar may have a trash function but I gues not. As for `xdg-config/kdeglobals` this was from Atom because they both are similar but this may not be necesary so I will remove that also.